app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -57,55 +58,56 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
 
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_talk_topic)
-        setSupportActionBar(replyToolbar)
+        binding = ActivityTalkTopicBinding.inflate(LayoutInflater.from(this))
```

Comment:
Can we just use the Activity's `getLayoutInflater()` function?

Reply:
done,